lib/Ray/sources/Utils/Cache.hpp

```diff
@@ -60,11 +60,13 @@ namespace RAY {
 			{};
 			std::shared_ptr<::ModelAnimation> fetch(const std::string &path, int *counter)
 			{
+				::ModelAnimation *animations = this->_dataLoader(path.c_str(), counter);
```

Comment:
That's a big no, you load the resource even if the resource is already in the cache.

Reply:
didn't saw that was changed during a valgrind fix
fixed